>å·¥ä½œè®¡åˆ’æ—¶é—´çº¿ï¼š  
2022.07.22-2022.08.10å°†å¦‚ä¸‹æœ‰å…³èƒ¶åŸè›‹ç™½æ¼”åŒ–çš„å·¥ä½œå®Œæˆ;  
2022.08.10-2022.08.20ä»ä¸Šè¿°å·¥ä½œç»“æœä¸­æ¢è®¨å…¶å®ƒæ–¹é¢ã€‚

# ä¸€ã€çµé•¿ç›®ä¸­IVå‹èƒ¶åŸåŸºå› çš„æ¼”åŒ–ï¼ˆå‚è€ƒè‚ŒåŠ¨è›‹ç™½æ¼”åŒ–çš„æ ‡å‡†æ¼”åŒ–æ–¹æ³•ï¼‰
## 1ã€ç¡®å®šç ”ç©¶å¯¹è±¡åŠåŸºå› 
+ ç ”ç©¶å¯¹è±¡ï¼š
```
7ä¸ªçµé•¿ç›®ç‰©ç§ï¼š
human (Homo sapiens)  
chimpanzee (Pan troglodytes)  #é»‘çŒ©çŒ©
gorilla (Gorilla gorilla)  #å¤§çŒ©çŒ©
Sumatran orangutan (Pongo abelii)  #è‹é—¨ç­”è…ŠçŒ©çŒ©
gibbon (Nomascus leucogenys) #é•¿è‡‚çŒ¿ 
crab-eating macaque (Macaca fascicularis) #é•¿å°¾çŒ•çŒ´
White-tufted-ear marmoset (Callithrix jacchus)  #æ™®é€šç‹¨çŒ´
```
+ ç¡®å®šIVå‹èƒ¶åŸåŸºå› 
```
åœ¨ensembleå®˜ç½‘è¿›å…¥ç›¸åº”ç‰©ç§ç•Œé¢ï¼Œæœç´¢COL4,ç‚¹å‡»åŸºå› ï¼Œå»æ‰ncRNAã€å‡åŸºå› ã€ç»“åˆè›‹ç™½ä¹‹ç±»åï¼ŒæŸ¥çœ‹å¾—ï¼š
7ä¸ªç‰©ç§å‡æœ‰6ä¸ªIVå‹èƒ¶åŸåŸºå› ,:COL4A1 COL4A2 COL4A3 COL4A4 COL4A5 COL4A6
```
## 2ã€å¤šåºåˆ—æ¯”å¯¹å’Œå»ºæ ‘  
ï¼ˆæ­¤æ­¥æˆ‘è¾“å…¥MEGAçš„æ–‡ä»¶æ˜¯CRLFï¼Œå¤šåºåˆ—æ¯”å¯¹å®Œä¿å­˜çš„æ˜¯.megæ ¼å¼è¿›è¡Œçš„åç»­åˆ†æï¼‰
+ ä¸‹è½½ã€æ•´ç†7ä¸ªç‰©ç§IVå‹èƒ¶åŸè›‹ç™½åŸºå› çš„æ°¨åŸºé…¸åºåˆ—å’ŒCDSåºåˆ— 
```
ä¸‹è½½ï¼šè¿›å…¥ensembleå®˜ç½‘ä¸‹é€‰ç›¸åº”ç‰©ç§ï¼Œæ¯ä¸ªåŸºå› æš‚å–æœ€é•¿çš„è½¬å½•æœ¬å¯¹åº”çš„è›‹ç™½åºåˆ—å’ŒCDSåºåˆ—ï¼Œæ‰‹åŠ¨ä¸‹è½½ã€‚ï¼ˆå…¶å®æˆ‘æƒ³ç”¨å‘½ä»¤è¡Œé€šè¿‡FTPä¸‹è½½å†ä¿ç•™æœ€é•¿è½¬å½•æœ¬å†æå–ï¼Œä½†ä¸€æ—¶æ²¡æƒ³å¥½æ€ä¹ˆç”¨å‘½ä»¤è¡Œå®Œæˆï¼Œæš‚ä¸”æ‰‹åŠ¨ä¸‹è½½ï¼‰ 
æ³¨ï¼šä¸‹è½½è¿‡ç¨‹ä¸­ï¼Œæœç´¢GorillaåŸºå› æ—¶Only searching Western Lowland Gorilla; æœç´¢GibbonåŸºå› æ—¶Only searching Northern white-cheeked gibbon  
æ·»åŠ åç¼€ï¼šä¸ºæ¯ä¸ªç‰©ç§ä¸­çš„åºåˆ—åæ·»åŠ åç¼€ï¼Œè§„åˆ™æ˜¯å­¦åçš„ä¸‰ä¸ªå­—æ¯ï¼Œå¦‚äººç”¨Hsaä½œä¸ºåç¼€;cdsåºåˆ—åœ¨åç¼€å‰åŠ _cdsç”¨ä»¥ä¸æ°¨åŸºé…¸åºåˆ—åŒºåˆ†
åˆå¹¶åºåˆ—ï¼šä»¥åˆå¹¶7ä¸ªç‰©ç§æ°¨åŸºé…¸åºåˆ—ä¸ºä¾‹ï¼š  
JOB=$(ls)
 for i in $JOB ; do cat $i >> protein.all.fa ; done  
 cat protein.all.fa | grep ">" | wc -l #42  
```
+ æ°¨åŸºé…¸ä¸cdsåºåˆ—çš„å¤šåºåˆ—æ¯”å¯¹åŠå»ºæ ‘
```
ä½¿ç”¨muscleå¤šåºåˆ—æ¯”å¯¹ï¼Œä¿å­˜ä¸ºmegaæ ¼å¼ å‚æ•°é»˜è®¤  
NJå»ºæ ‘ å‚æ•°é»˜è®¤ å»ºå‡ºçš„æ ‘COL4A5ç¡®å®ä¸COL4A1æŒºè¿‘ æ ‘çš„PDFæ–‡ä»¶è§æ–‡ä»¶å¤¹MEGAå†…çš„proteinå’Œcdsæ–‡ä»¶å¤¹
æ°¨åŸºé…¸åºåˆ—å¾—åˆ°çš„Original Treeè§ä¸‹å›¾1ï¼šæœ‰ä¸ªå°ç–‘é—®ï¼šæ°¨åŸºé…¸åºåˆ—å¾—åˆ°çš„Original Treeç›´æ¥ä»MEGAå¯¼å‡ºæ—¶æ˜¯ä¸¤å¤§åˆ†æ”¯ï¼Œåœ¨iTOLæ˜¯ä¸‰ä¸ªå¤§åˆ†æ”¯ï¼Œä¸ºä»€ä¹ˆï¼Ÿ
```
![Original Tree](./iRtOOle2jbw5HH18GvcYfA.svg)  
å›¾1â€”â€”æ°¨åŸºé…¸åºåˆ—å¾—åˆ°çš„Original Tree
+ åˆ†ç»„
```
å¾ˆæ˜æ˜¾ï¼Œå¯ä»¥åˆ†ä¸º6ä¸ªç›´ç³»åŒæºç»„ï¼ˆæ¯ä¸ªç‰©ç§çš„åŸºå› å‡ºç°ä¸€æ¬¡ï¼‰ï¼Œæ²¡æœ‰complexç»„ï¼ˆä¸€ä¸ªç‰©ç§çš„ä¸€ä¸ªåŸºå› å¤šæ‹·è´æˆ–å°‘äºä¸€ä¸ªæ‹·è´ï¼‰.
```
+ è®¡ç®—Ka Kså’ŒÎ 
```
1.  
è®¡ç®—ka ksï¼ˆ6ä¸ªç›´ç³»åŒæºç»„ï¼Œæ‰€ä»¥åˆ†6æ¬¡è®¡ç®—ï¼Œæ¯æ¬¡é‡Œåˆ†åˆ«ç®—ka ksï¼‰  
æ•´ç†6ä¸ªç›´ç³»åŒæºç»„çš„åºåˆ—
 cat cds.all.fa | grep "COL4A1" | cut -d ">" -f 2 >COL4A1.lst  
 faops some cds.all.fa COL4A1.lst ./COL4A1/COL4A1.fa
   
cat cds.all.fa | grep "COL4A2" | cut -d ">" -f 2 >COL4A2.lst  
 faops some cds.all.fa COL4A2.lst ./COL4A2/COL4A2.fa

cat cds.all.fa | grep "COL4A3" | cut -d ">" -f 2 >COL4A3.lst  
 faops some cds.all.fa COL4A3.lst ./COL4A3/COL4A3.fa

cat cds.all.fa | grep "COL4A4" | cut -d ">" -f 2 >COL4A4.lst  
 faops some cds.all.fa COL4A4.lst ./COL4A4/COL4A4.fa

cat cds.all.fa | grep "COL4A5" | cut -d ">" -f 2 >COL4A5.lst  
 faops some cds.all.fa COL4A5.lst ./COL4A5/COL4A5.fa

cat cds.all.fa | grep "COL4A6" | cut -d ">" -f 2 >COL4A6.lst  
 faops some cds.all.fa COL4A6.lst ./COL4A6/COL4A6.fa  

å¤šåºåˆ—æ¯”å¯¹ï¼Œä½¿ç”¨MUSCLE å‚æ•°é»˜è®¤  
ä½¿ç”¨MEGAè®¡ç®—Ka Ksï¼Œæ­¥éª¤è§Dynamic Actin Gene Family Evolution in Primates.md  
ç»“æœè§Table1å’Œsupplement.table  
ç–‘é—®ï¼šæ–‡ç« Dynamic Actin Gene Family Evolution in Primatesï¼ˆä»¥ä¸‹ç®€ç§°è‚ŒåŠ¨è›‹ç™½æ–‡ç« ï¼‰çš„Table1çš„ka/ksæ˜¯æ€ä¹ˆç®—å‡ºçš„ï¼Œä¼¼ä¹ä¸æ˜¯é™„è¡¨é‡Œka/ksçš„å¹³å‡å€¼ã€‚å¦å¤–è‚ŒåŠ¨è›‹ç™½æ–‡ç« ä¸­è¯´Whether in orthologous groups or
complex groups, the average ğ¾ğ‘/ğ¾ğ‘  ratios of most groups(82.4%) are much smaller than 1 (only six groups of average ğ¾ğ‘/ğ¾ğ‘  ratios are greater than 0.5, all of them belong to
complex groups), indicating that the actin genes code highly conserved proteins because of important functions and were under strong negative selection.ä»ğ¾ğ‘/ğ¾ğ‘  ratioså¯ä»¥çœ‹å‡ºé€‰æ‹©å‹åŠ›æˆ‘ç†è§£ï¼Œä½†æ€ä¹ˆindicatingè‚ŒåŠ¨è›‹ç™½åŸºå› ç¼–ç é«˜åº¦ä¿å®ˆçš„è›‹ç™½å‘¢ï¼Ÿ  

2.
è®¡ç®—Î 
è§æ•™ç¨‹https://blog.csdn.net/qq_50637636/article/details/122612457  
ä½¿ç”¨çš„å‚æ•°è§å›¾2  
æœ‰ä¸€ä¸ªå°ç–‘é—®ï¼šMEGAç»™å‡ºÎ çš„ç»“æœæ˜¯åªæœ‰å¯¹è§’çº¿æ²¡æœ‰å€¼ï¼Œä½†å¯¼å‡ºä¸ºexcelåï¼Œå°‘äº†ä¸Šé¢ä¸€éƒ¨åˆ†å€¼ï¼Œå…¶å®æˆ‘ä¸ªäººè®¤ä¸ºä¸¤ä¸ªåºåˆ—ä¹‹é—´çš„æ ¸è‹·é…¸å·®å¼‚å’Œè¿™ä¸¤ä¸ªåºåˆ—å‰åæ²¡æœ‰å…³ç³»ï¼Œåºåˆ—1ä¸åºåˆ—2çš„å·®å¼‚å’Œåºåˆ—2ä¸åºåˆ—1çš„å·®å¼‚åº”è¯¥ä¸€æ ·æ‰æ˜¯ï¼Œæ‰€ä»¥æ²¡ä¸Šé¢çš„ç»“æœä¹Ÿæ­£å¸¸ï¼Œçœ‹äº†ä¸¤ä¸ªè‚ŒåŠ¨è›‹ç™½æ–‡ç« é™„è¡¨2çš„ä¸¤ä¸ªåºåˆ—çš„PIï¼Œåºåˆ—é¡ºåºä¸åŒï¼Œç»“æœåŒã€‚ 
æˆ‘å¾—å‡ºçš„Î çš„ç»“æœè§Evolution_/IV_collagenä¸‹çš„Table1å’Œsupplement.table 
```


![](./MEGA/Î /è®¡ç®—Î æ—¶çš„å‚æ•°.png)
![](./MEGA/Î /MEGAä¸Šçœ‹åˆ°çš„Î ç»“æœ.png)  
å›¾3â€”â€”MEGAä¸Šçœ‹åˆ°çš„Î ç»“æœ
![](./MEGA/Î /å¯¼å‡ºä¸ºexcelåçœ‹åˆ°çš„ç»“æœ.png)  
å›¾4â€”â€”Î å¯¼å‡ºä¸ºexcelåçœ‹åˆ°çš„ç»“æœ  

## 3ã€é‰´å®šIVå‹èƒ¶åŸå‡åŸºå› 
## 4ã€å¯†ç å­ä½¿ç”¨æƒ…å†µ
ä½¿ç”¨åœ¨çº¿ç½‘ç«™ä½¿ç”¨åœ¨çº¿ç½‘ç«™è®¡ç®—RSCUï¼Œç½‘å€ï¼šhttp://cloud.genepioneer.com:9929/#/tool/alltool/detail/214
åœ¨çº¿ç½‘ç«™è¿”å›çš„ç»“æœlog.txté‡Œæ˜¾ç¤º start codon is wrong(ATG)ï¼Œä½†ç”±äºåç»­ä¼šæŠŠèµ·å§‹å¯†ç å­åˆ å»ï¼Œæ‰€ä»¥æš‚ä¸è®¤ä¸ºæ­¤é”™è¯¯æœ‰å½±å“ã€‚
å°†ç»“æœæ•´ç†ä¸ºtsvæ–‡ä»¶ï¼ˆå…¨é€‰åœ¨çº¿ç½‘ç«™è§£å‹å‡ºçš„excelçš„å†…å®¹ï¼Œç²˜è´´åˆ°æ–°å»ºçš„çš„LFçš„tsvæ–‡ä»¶ã€‚ä¹‹æ‰€ä»¥è¦ç”¨LFæ˜¯å› ä¸ºtsv-filterå¯¹CRLFçš„æ–‡ä»¶ä¼šæŠ¥é”™ã€‚æ³¨æ„åœ¨VSCodeæ‰“å¼€ä¸€ä¸ªç©ºtsvæˆ–åªæœ‰ä¸€è¡Œçš„tsvï¼Œå°±ç®—æ”¹æˆLFä¿å­˜å†æ‰“å¼€ä»æ˜¯CRLFï¼Œä½†æœ‰ä¸¤è¡Œæ–‡å­—æ—¶ç”±CRLFæ”¹æˆLFä¿å­˜å†æ‰“å¼€å³ä¸ºLF.ï¼‰  
å¯¹äººå…­ä¸ªIVå‹èƒ¶åŸè›‹ç™½åŸºå› çš„cdsè¿›è¡ŒRSCUåˆ†æåï¼Œç”¨åœ¨çº¿è½¯ä»¶ç”Ÿæˆçƒ­å›¾ï¼Œè§å›¾5.  
![](./rscu_heatmap.png)  
å›¾5


+ å¯¹æ•°æ®è¿›è¡Œé¢„å¤„ç†ï¼ˆåˆ é™¤ç»ˆæ­¢å¯†ç å­å’ŒATGã€TGGï¼‰
```
cd RSCU
JOB=$(find ./ -maxdepth 2 -type f -name "*.tsv")
for J in $JOB;do
  echo -e "====> $J"
  tsv-filter -H --str-ne Codon:UAA $J | 
  tsv-filter -H --str-ne Codon:UAG | 
  tsv-filter -H --str-ne Codon:UGA |
  tsv-filter -H --str-ne Codon:AUG |
  tsv-filter -H --str-ne Codon:UGG > tem&&
  mv tem $J
done
```
+ åˆå¹¶
```
cd within_groups/
JOB=$(ls)
tsv-select -H --fields Codon o1.tsv > merge.tsv
for J in $JOB;do
  echo -e "===> $J"
  tsv-join --filter-file $J -H --key-fields Codon --append-fields RSCU merge.tsv > tem
    mv tem merge.tsv
done
```
```
cd within_species/
JOB=$(ls)
tsv-select -H --fields Codon 0.tsv > merge.tsv
for J in $JOB;do
  echo -e "===> $J"
  tsv-join --filter-file $J -H --key-fields Codon --append-fields RSCU merge.tsv > tem
    mv tem merge.tsv
done
```
 
+ è®¡ç®—æ–¹å·®
ä½†æˆ‘ä¸ç¡®å®šæ–¹å·®å’Œè‚ŒåŠ¨è›‹ç™½æ–‡ç« ä¸­è¯´çš„variationä¸æ˜¯ä¸€ä¸ªæ„æ€ã€‚æ­£åœ¨ç¢ç£¨ã€‚æš‚æ—¶ä»¥æ–¹å·®è¿›è¡Œè®¡ç®—å’Œç”»å›¾ã€‚
```
#åˆ©ç”¨datamashä¸­çš„svarè®¡ç®—(brew install datamash)
datamash --help

# æŸ¥çœ‹çŸ©é˜µ
head merge.tsv

# è®¡ç®—
cat merge.tsv | datamash transpose | datamash --header-in --header-out svar 2-60 | datamash transpose > svar.tsv
# transpose è½¬ç½®
# --header-in           first input line is column headers
# --header-out          print column headers as first line
```
+ Rç”»å›¾   
within_groupså’Œwithin_speciesåˆ†å¼€ä½œå›¾ä»¥åŠåˆå¹¶ä¸ºç°‡çŠ¶æ¡å½¢å›¾éƒ½åœ¨ä¸‹æ–¹å†™äº†è¿‡ç¨‹ã€‚å±•ç¤ºçš„å›¾6æ˜¯ç°‡çŠ¶æ¡å½¢å›¾ã€‚
```
#within_groups
setwd("D:/0~GitHub/Evolution_/IV_collagen/RSCU/within_groups")
group_svar <- read.table("svar_within_groups.tsv",header=FALSE,sep='\t')
library(ggplot2)
p <- ggplot(group_svar,aes(V1,V2))+geom_bar(stat = 'identity')+ylab("variance of RSCU within groups") +xlab("Codons")+ theme(panel.grid = element_blank())
p

#within_species
setwd("D:/0~GitHub/Evolution_/IV_collagen/RSCU/within_species")
species_svar <- read.table("svar_within_species.tsv",header=FALSE,sep='\t')
library(ggplot2)
p <- ggplot(species_svar,aes(V1,V2))+geom_bar(stat = 'identity')+ylab("variance of RSCU within species") +xlab("Codons")+ theme(panel.grid = element_blank())
p
```
```
ç”»ç°‡çŠ¶æ¡å½¢å›¾
#æ•´ç†æ•°æ®
cd R #è¿›å…¥æ–°æ–‡ä»¶å¤¹
(echo -e "Codons\tvariance_of_RSCU_within_groups" && cat svar_within_groups.tsv) > tem&&
    mv tem svar_within_groups.tsv # æ·»åŠ è¡¨å¤´ 
#æ‰‹åŠ¨åˆ›å»ºcultivar.tsv
paste cultivar.tsv svar_within_groups.tsv >svar_within_groups_paste.tsv #æŒ‰åˆ—åˆå¹¶ï¼Œä¹Ÿç›¸å½“äºå¢åŠ æ–°åˆ—

(echo -e "Codons\tvariance_of_RSCU_within_species" && cat svar_within_species.tsv) > tem&&
    mv tem svar_within_species.tsv # æ·»åŠ è¡¨å¤´ 
#æ‰‹åŠ¨åˆ›å»ºcultivar.tsv
paste cultivar.tsv svar_within_species.tsv >svar_within_species_paste.tsv

#åˆå¹¶ï¼ŒæŒ‰è¡Œè¿½åŠ  åˆ°RSCUç›®å½•
cat svar_within_species_paste.tsv >> svar_within_groups_paste.tsv
æ‰‹åŠ¨åˆ ä¸­é—´è¡¨å¤´
mv svar_within_groups_paste.tsv svar_within_groups_species.tsv
setwd("D:/0~GitHub/Evolution_/IV_collagen/RSCU")
svar <- read.table("svar_within_groups_species.tsv",header=TRUE,sep='\t')
library(ggplot2)
ggplot(svar, aes(x = Codons, y = variance_of_RSCU_within_groups, fill = Cultivar)) +geom_bar(position = "dodge", stat = "identity")+ theme(panel.grid = element_blank())
#è§å›¾6,å¯è§ç‰©ç§é—´æœ‰ç›¸ä¼¼çš„å¯†ç å­ä½¿ç”¨æ¨¡å¼ï¼Œç›¸æ¯”èµ·æ¥ç›´ç³»åŒæºç»„é—´ä½¿ç”¨æ¨¡å¼æ›´å¤šæ ·åŒ–ã€‚
```
![å›¾6](./RSCU/Figure1_variations_of_RSCU.jpeg)
å›¾6 variations of RSCU
## 5ã€IVå‹èƒ¶åŸåŸºå› è¡¨è¾¾æ¨¡å¼


å¤§æ¦‚ç‡ä¸ä¼šæœ‰complexç»„ï¼Œå¯èƒ½æ‰¾ä¸åˆ°å‡åŸºå› ã€‚å‡è‹¥æœ‰å‡åŸºå› ï¼Œç”±å‡åŸºå› æ¨æ ¸è‹·é…¸å†…ç§‰çªå˜ç‡ã€‚

# äºŒã€IVå‹èƒ¶åŸä¸­G-X-Yçš„æ¼”åŒ–æ¢ç©¶ï¼ˆåœ¨æ ‡å‡†æ¼”åŒ–ä¹‹å¤–çš„ç‰¹å¾æ¼”åŒ–æ–¹é¢ï¼‰
## æ—ç³»åŒæºä¸­GXY
å°†äººçš„6ä¸ªIVå‹èƒ¶åŸåŸºå› çš„æ°¨åŸºé…¸åºåˆ—å’Œæ ¸è‹·é…¸åºåˆ—è¿›è¡Œå¤šåºåˆ—æ¯”å¯¹åå»ºæ ‘ï¼ŒæŸ¥çœ‹è¿›åŒ–å…³ç³»ã€‚
æŸ¥çœ‹æ¯ä¸ªåŸºå› ä¸­çš„ä¸­æ–­ï¼Œçœ‹å“ªäº›è¾ƒä¸ºå¤è€ï¼Œå“ªäº›è¾ƒä¸ºæ–°å‡ºã€‚å°†CLO4A5åŸºå› å‡ºç°çš„ä¸­æ–­åˆ†ç±»ï¼šä¸€äº›COL4A5ç‹¬æœ‰çš„ã€åœ¨6ä¸ªåŸºå› ä¸­éƒ¨åˆ†æœ‰ã€åœ¨6ä¸ªåŸºå› ä¸­éƒ½æœ‰çš„ï¼ˆæ­¤ç±»è¾ƒä¸ºä¿å®ˆï¼‰
## ç›´ç³»åŒæºä¸­GXY
ç”¨ç»Ÿè®¡åˆ†æéƒ¨åˆ†æ‰€é€‰çš„6ä¸ªç­‰çº§30ä¸ªç‰©ç§ï¼ˆçµé•¿ç›®åˆ°åŒ—æ–¹å…½ç±»ï¼‰çš„COL4A5åŸºå› çš„æ°¨åŸºé…¸åºåˆ—å¤šåºåˆ—æ¯”å¯¹ï¼Œæ¯”è¾ƒä¸­æ–­ã€‚å°†äººç§‘ä½œä¸ºå‚ç…§ï¼Œçœ‹å“ªäº›ä¸­æ–­æ˜¯å¤è€æ˜¯å“ªäº›æ˜¯æ–°çš„ï¼ˆäººæœ‰è€Œå…¶å®ƒç‰©ç§æ²¡æœ‰çš„å¯èƒ½æ˜¯æ–°çš„ï¼Œäººæœ‰è€Œå…¶å®ƒç‰©ç§æœ‰çš„å¯èƒ½æ˜¯è€çš„ï¼‰ï¼Œäººä¸ä¸­æ–­å…¶å®ƒç‰©ç§ä¸­æ–­çš„ä½ç½®å¯èƒ½ä¸é‡è¦ï¼Œå› ä¸ºå…è®¸åœ¨å…¶å®ƒç‰©ç§ä¸­ä¸­æ–­ã€‚
ä¸­æ–­çš„é•¿åº¦å’Œä½ç½®ä¹Ÿéœ€è¦è€ƒè™‘ï¼Œå› ä¸ºThe location of interruptions is often similar for the three chains within a heterotrimer, while there may be different lengths of the interruptions at a given site in each of the three chains.[Thiagarajan G, Li Y, Mohs A, et al. Common interruptions in the repeating tripeptide sequence of non-fibrillar collagens: sequence analysis and structural studies on triple-helix peptide models.Â J Mol Biol. 2008;376(3):736-748. doi:10.1016/j.jmb.2007.11.075]

# å¯é€‰â€”â€”è€ƒè™‘å…¶å®ƒåŸºå› å®¶æ—çš„æ¼”åŒ–ï¼ˆæ¯”å¦‚å°†è‚ŒåŠ¨è›‹ç™½åŸºå› ä½œä¸ºå¯¹æ¯”ï¼‰
